### ISSUE TITLE
AS pointer in the RD

### ISSUE THREAD

**Issue by erdtman** [ace-wg/ace-oauth#139]
Review comment from Jim.

Section "5.1.  Discovering Authorization Servers" second paragraph implies that there is an AS pointer registered in the RD, that attribute is not registered. And it is not described how to to work with the RD to find AS.

We should either remove this implication or add more details.

**Comment by LudwigSeitz** [comment]
We could rephrase this to say this instead:

"Instead of the initial Unauthorized Resource Request message, other discovery methods may be used, or the client may be pre-provisioned with the address of the AS.
